ui/shared/components/panel/variants/Pathogenicity.jsx

```diff
@@ -34,16 +33,20 @@ const HGMD_CLASS_NAMES = {
 }
 const hgmdName = hgmdClass => HGMD_CLASS_NAMES[hgmdClass]
 
+const ClinvarStars = ({ goldStars }) => goldStars != null &&
+  <StarsContainer>
+    {Array.from(Array(4).keys()).map(i => (i < goldStars ? <StarIcon key={i} goldstar="yes" /> : <StarIcon key={i} />))}
```

Comment:
Why not just make `goldstar` a boolean prop?

Reply:
It causes this warning
https://github.com/styled-components/styled-components/issues/1198

Reply:
You need to do `const StarIcon = styled(({ goldstar, ...props }) => <Icon {...props} />).attrs({ name: 'star' })``

Reply:
theres also a `ColoredIcon` component in `shared/components` that takes any valid CSS color for the color prop